app/models/user.rb

```diff
@@ -1,6 +1,11 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] One of the project's requirements is that users should confirm their email before they can navigate the application. Kindly add `:confirmable` to the attributes and implement the feature for confirming email. 👍